### ISSUE TITLE
Requirement is pinned for openwrt-luci-rpc 1.1.3: requests==2.21.0

### ISSUE THREAD

**Issue by MartinHjelmare** [fbradyirl/openwrt-luci-rpc#43]
* openwrt-luci-rpc version: 1.1.3
* Python version: 3.7
* Operating System: Ubuntu 18.04

### Description

- This library supports an integration in Home Assistant.
- The upcoming pip resolver will be more strict in resolving package version conflicts:
http://pyfound.blogspot.com/2020/03/new-pip-resolver-to-roll-out-this-year.html#what-will-change
- To avoid future issues it would be good if this library would change its pinning to allow any higher version and just set a minimum version.

Thanks for your support of Home Assistant!

### What I Did

```
(home-assistant) x:~/Dev/home-assistant/home-assistant$ pip install -r requirements_all.txt
...
(home-assistant) x:~/Dev/home-assistant/home-assistant$ pipdeptree --reverse --packages requests
...
requests==2.24.0
...
  - openwrt-luci-rpc==1.1.3 [requires: requests==2.21.0]
```

**Comment by onkelbeh** [comment]
*ping*

**Comment by fbradyirl** [comment]
Fixed in version 1.1.5
